recipe/patches/0015-Add-CondaEcosystemModifyDllSearchPath.patch

```diff
@@ -1,7 +1,48 @@
-From 9eb3eea71dcbe3c9c2649169bd35614998b1c650 Mon Sep 17 00:00:00 2001
+From 5bb1915898462b89ea9ed9791e26a2e7e3c63fd2 Mon Sep 17 00:00:00 2001
 From: Ray Donnelly <mingw.android@gmail.com>
 Date: Tue, 24 Dec 2019 18:37:17 +0100
-Subject: [PATCH 20/30] Add CondaEcosystemModifyDllSearchPath()
+Subject: [PATCH] Add CondaEcosystemModifyDllSearchPath()
+
+There are 2 modes depending on CONDA_DLL_SEARCH_MODIFICATION env variable
+
+- unset CONDA_DLL_SEARCH_MODIFICATION (Default)
+
+  In this mode, the python interpreter works as if the python interpreter
+  was called with the following directories.
+
+    os.add_dll_directory(join(sys.prefix, 'bin'))
+    os.add_dll_directory(join(sys.prefix, 'Scripts'))
+    os.add_dll_directory(join(sys.prefix, 'Library', 'bin'))
+    os.add_dll_directory(join(sys.prefix, 'Library', 'usr', 'bin'))
+    os.add_dll_directory(join(sys.prefix, 'Library', 'mingw-w64', 'bin'))
+
+  Search order
+    - The directory that contains the DLL
+    - Application directory
+    - Directories added with os.add_dll_directory
+    - Above 5 directories
+    - C:\Windows\System32
```

Comment:
```suggestion
  Search order
    - The directory that contains the DLL
    - Application directory
    - Directories added with os.add_dll_directory
    - Above 3 directories
    - C:\Windows\System32
```

Reply:
It's not the 3 directories directly above. The 5 directories mentioned earlier in the comment